src/swift/Sources/APIViewManager.swift

```diff
@@ -89,11 +89,35 @@ class APIViewManager {
         for statement in decl.statements {
             switch statement {
             case let decl as ClassDeclaration:
-                process(decl)
-            case let decl as StructDeclaration:
-                process(decl)
+              process(decl)
+            case let decl as ConstantDeclaration:
+              process(decl)
+            case let decl as DeinitializerDeclaration:
+              process(decl)
             case let decl as EnumDeclaration:
-                process(decl)
+              process(decl)
+            case let decl as ExtensionDeclaration:
+              process(decl)
+            case let decl as FunctionDeclaration:
+              process(decl)
+            case let decl as ImportDeclaration:
+              process(decl)
+            case let decl as InitializerDeclaration:
+              process(decl)
+            case let decl as OperatorDeclaration:
+              continue // process(decl)
+            case let decl as PrecedenceGroupDeclaration:
+              continue // process(decl)
+            case let decl as ProtocolDeclaration:
+              process(decl)
+            case let decl as StructDeclaration:
+              process(decl)
+            case let decl as SubscriptDeclaration:
+              continue // process(decl)
+            case let decl as TypealiasDeclaration:
+              process(decl)
+            case let decl as VariableDeclaration:
+              process(decl)
             default:
                 continue
             }
```

Comment:
You can also just do a case-let to `Declaration` and then pass it along to the method you have for processing declarations to clean the up. However, I strongly recommend that these methods should live in the TokenFile, not the APIViewManager